target/scripts/startup/setup.d/security/spoofing.sh

```diff
@@ -6,9 +6,9 @@ function _setup_spoof_protection() {
 
     if [[ ${ACCOUNT_PROVISIONER} == 'LDAP' ]]; then
       if [[ -z ${LDAP_QUERY_FILTER_SENDERS} ]]; then
-        postconf 'smtpd_sender_login_maps = ldap:/etc/postfix/ldap-users.cf ldap:/etc/postfix/ldap-aliases.cf ldap:/etc/postfix/ldap-groups.cf'
+        postconf 'smtpd_sender_login_maps = ldap:/etc/postfix/ldap/users.cf ldap:/etc/postfix/ldap/aliases.cf ldap:/etc/postfix/ldap/groups.cf'
```

Comment:
I'm also not sure about this in general. I think it was the original approach before `senders.cf` was contributed, and was provided as a compatibility fallback. I can check.

The `result_attribute` for these configs isn't necessarily going to be correct for this feature like it should be for `senders.cf`.

A future PR can address that though.

Reply:
👍🏼